test/resource.test.js

```diff
@@ -0,0 +1,373 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+
+let accessToken = null;
+
+describe("POST Tests Resource Account",()=>{
+    it("POST /Resource Account Bad Request", (done) =>{
+        chai.request(server).post("/resources/account")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idAccount', '6164db5')
+        .attach('file', `${__dirname}/resources/imageAccount.png`, 'imageAccount.png')
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("POST /Resource Account Bad Request Account", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
    it("POST /Resource Account Bad Request Account", (done) => {
```